charters/DeiCharter.md

```diff
@@ -0,0 +1,35 @@
+# Diversity Equity and Inclusion Committee Charter
+
+Project Jupyter welcomes and encourages participation in our community by people of all backgrounds and identities. We are committed to promoting and sustaining a culture that values mutual respect, tolerance, and learning. We work together as a community to help each other live out these values.
+
+The Diversity Equity & Inclusion (DEI) Committee is a standing committee established under Jupyter’s governance model. The committee reviews its charter annually and may make changes at any time during the year. Substantive changes to the purpose or mission must be submitted to the Executive Council for approval.
+
+## WHY: Purpose 
+
+Why is the committee being created? How does this group contribute to Jupyter? 
+
+The purpose of this committee is to lead strategic initiatives designed to improve the diversity, equity, and inclusion of Project Jupyter community leaders, core developers, and end users across the globe. For the long-term sustainability of our community, we champion efforts to build an open source environment where people feel respected, valued, and encouraged to continue participating. 
```

Comment:
Should we include details about the scope of the DEI group, namely that it covers all contexts of Jupyter work: software, working groups, standing committees, JupyterCon, online forums, Github, EC, SSC, etc.?

Reply:
Agreed!